### ISSUE TITLE
REVIEWS.io Widgets blocked by DuckDuckGo Privacy

### ISSUE THREAD

**Issue by Ben-Bromley** [duckduckgo/duckduckgo-privacy-extension#1213]
## Description

Hi! I'm a developer for [REVIEWS.io](https://www.reviews.io/) and one of our clients has come to us to complain that the DuckDuckGo mobile application is preventing our widgets showing on their site, can anything be done about this?

Our widgets typically consist of an empty `<div>` with an ID, an imported Javascript file using `<script>`, and some initialisation script to run the JavaScript pulled in by the `<script>` tag. 

I'm not 100% on this but based on my understanding of DuckDuckGo, I believe the request for that script is being blocked

Script being imported:
`<script src="https://widget.reviews.io/polaris/build.js"></script>`

Most if not all of our widget scripts come from the widget.reviews.io subdomain. Could whitelisting that domain in the privacy protector be a valid option?

## Steps to Reproduce

1. Install DuckDuckGo Mobile App (I use iOS)
2. go to https://lovmate.com.au/pages/reviews
3. look at the page below the "Customer Reviews" title

**Expected behavior:**

REVIEWS.io Widget shows on screen (can be achieved by disabling privacy protection, not ideal)
<img src="https://user-images.githubusercontent.com/74667456/170981776-fbedfc6d-6311-4718-a7bc-99fe525a0092.png"
     alt="DuckDuckGo Mobile Showing Widget"
     width="200" />
     
**Actual behavior:**

Page content below "Customer Reviews" header is blank
<img src="https://user-images.githubusercontent.com/74667456/170981920-32691fdd-3c99-4c9d-8bef-8b6cfda1cecc.png"
     alt="DuckDuckGo Mobile NOT Showing Widget"
     width="200" />

**Versions**

- Browser: DuckDuckGo Mobile
- OS: iOS v15+

## Additional Information

 - This is a consistent issue
 - Disabling privacy protection does allow our widgets to show, but obviously that's not the default setting, and would result in many of our clients customers not seeing their reviews, which is not good.

**Comment by crwils** [comment]
Did you find a resolution to this? I'm having similar issues with a Trustpilot widget.

**Comment by sammacbeth** [comment]
`widget.reviews.io` was removed from our blocklist back in June, so this should be resolved.

@crwils if you have an issue with another widget, could you open a separate issue for it, including the URL of the blocked widget?
